packages/hardware-ledger/src/device-connection.ts

```diff
@@ -59,14 +59,17 @@ export type OpenLedgerConnectionInput = Readonly<{
 
 const __openConnection = async (
 	isLoggingEnabled: boolean,
+	transport: BasicLedgerTransport,
 	input?: OpenLedgerConnectionInput,
 ): Promise<BasicLedgerTransport> => {
 	if (isLoggingEnabled) {
 		log.debug(`🔌⏱ Looking for (unlocked 🔓) Ledger device to connect to.`)
 	}
 
+	/*
 	const basicLedgerTransport: BasicLedgerTransport = await import(
-		'@ledgerhq/hw-transport-node-hid'
+		// @ts-ignore
+		'@aleworm/hw-transport-node-hid'
```

Comment:
maybe some note about `ts-ignore` and note about why we are using a custom package?

Reply:
this isn't used